library/src/compute/construct.rs

```diff
@@ -184,6 +186,185 @@ cast_from_value! {
     },
 }
 
+/// Create a new datetime.
```

Comment:
I think we should have links between this docs page and the type page, in both directions.

Reply:
Should be there now.